src/client/java/io/wispforest/limelight/impl/config/LimelightTheme.java

```diff
@@ -73,4 +93,8 @@ static LimelightTheme current() {
     int sourceExtensionColor();
 
     int resultEntryTextColor();
+
+    int expandedResultChildEntryBackgroundColor();
+
+    int expandedResultChildEntrySourceExtensionColor();
```

Comment:
these method names are a bit long
i'm hoping to add custom theme support via config at some point and having such long config keys might be bad

Reply:
Fair, I agree, any ideas of good names? maybe `childEntryBackgroundColor` and `childEntrySourceExtentionColor`? might still be too long. Though if all methods are colors, maybe the term color could be removed from all of them?